hedera-node/src/main/java/com/hedera/services/state/submerkle/ExpirableTxnRecord.java

```diff
@@ -537,6 +544,9 @@ public TransactionRecord asGrpc() {
 		if (hbarAdjustments != null) {
 			grpc.setTransferList(hbarAdjustments.toGrpc());
 		}
+		if (stakingRewardsPaid != null) {
+			grpc.addAllPaidStakingRewards(stakingRewardsPaid.toGrpc().getAccountAmountsList());
```

Comment:
It would be good if we had a `List<AccountAmount> CurrencyAdjustments.asAccountAmountsList()` method that avoids creating an intermediate `TransferList` object. Not urgent though.